test/fw/ptl/lib/pbs_api_to_cli.py

```diff
@@ -253,6 +253,7 @@
     ATTR_sched_cycle_len: 'sched_cycle_length',
     ATTR_do_not_span_psets: 'do_not_span_psets',
     ATTR_soft_time: 'Wsoft_limit_time',
+    ATTR_tolerate_node_failures: 'Wtolerate_node_failures=',
```

Comment:
Why is the '=' part of the string?

Reply:
This is the part of the PTL framework that maps an attribute to how it is set via qalter. The feature did not introduce a new option in qalter so it maps to '-Wtolerate_node_failures=' with the specified attribute \<value> tacked onto it. Basically, it maps to qalter -W tolerate_node_failures=\<value>.